onnxruntime/core/optimizer/constant_folding.h

```diff
@@ -16,8 +16,9 @@ it statically computes parts of the graph that rely only on constant initializer
 */
 class ConstantFolding : public GraphTransformer {
  public:
-  ConstantFolding(const std::unordered_set<std::string>& compatible_execution_providers = {}) noexcept
-      : GraphTransformer("ConstantFolding", compatible_execution_providers) {}
+  ConstantFolding(const std::unordered_set<std::string>& compatible_execution_providers = {},
+                  const std::unordered_set<std::string>& excluded_initializers = {}) noexcept
```

Comment:
>excluded_initializers [](start = 57, length = 21)

First look of this input is strange... 
but latter I realized that this is for avoid folding the weights_to_traini 

Let's have a comment explaining this

Reply:
Added.